### ISSUE TITLE
Error with text wrapping

### ISSUE THREAD

**Issue by aaronhowser1** [VazkiiMods/Patchouli#542]
### Mod loader

Forge

### Minecraft version

1.18.2

### Patchouli version

Patchouli-1.18.2-67

### Modloader version

Forge 39.0.64

### The latest.log file

Too big, I'll put the file lower down

### Issue description

Using this page:

```json
        {
            "type":"patchouli:spotlight",
            "item":"laserio:laser_wrench",
            "text":"The Laser Wrench allows you to connect Nodes, allowing their Cards to interact.$(br2)Sneak right-click one Node with the Wrench, then right-click another to connect them.$(br2)Two Nodes that both connect to a third, but not each other, still count as connected."
        },
```

makes it look like this, which is mostly what I want:
![image](https://user-images.githubusercontent.com/15698722/170793288-e27d0940-8f95-44a6-a3f4-142372e18730.png)

However, if I color the Laser Wrench text like this:
```json
        {
            "type":"patchouli:spotlight",
            "item":"laserio:laser_wrench",
            "text":"The $(item)Laser Wrench$() allows you to connect Nodes, allowing their Cards to interact.$(br2)Sneak right-click one Node with the Wrench, then right-click another to connect them.$(br2)Two Nodes that both connect to a third, but not each other, still count as connected."
        },
```

the text wrapping breaks
![image](https://user-images.githubusercontent.com/15698722/170793439-4eb4fb55-1602-4c58-ac84-fd37ed79efba.png)

### Other information

[latest.log](https://github.com/VazkiiMods/Patchouli/files/8789902/latest.log)

**Comment by aaronhowser1** [comment]
Oh it might be the squishing actually? I moved the final paragraph to a different page, and it's working fine now
![image](https://user-images.githubusercontent.com/15698722/170793592-c91f05a4-53b9-4738-b1b3-7d4cfe0717d2.png)

**Comment by Aizistral** [comment]
Yep, it's squishing because default text overflow management mode tries to compress the text if it doesn't fit into page. I already suggested to grant us the ability to fix that per book in #506.
